website/docs/r/encryption_at_rest.html.markdown

```diff
@@ -71,6 +71,8 @@ resource "mongodbatlas_encryption_at_rest" "test" {
 * `secret_access_key` - The IAM secret access key with permissions to access the customer master key specified by customerMasterKeyID.
 * `customer_master_key_id` - The AWS customer master key used to encrypt and decrypt the MongoDB master keys.
 * `region` - The AWS region in which the AWS customer master key exists: CA_CENTRAL_1, US_EAST_1, US_EAST_2, US_WEST_1, US_WEST_2, SA_EAST_1
+* `role_id` - ID of an AWS IAM role authorized to manage an AWS customer master key.
```

Comment:
The API doc notes "ID of an AWS IAM role authorized to manage an AWS customer master key. To find the ID for an existing IAM role, send a GET request to the cloudProviderAccess API endpoint."   

We should probably phrase this a bit the same, I'd suggestion something like: 
"ID of an AWS IAM role authorized to manage an AWS customer master key.  To find the ID for an existing IAM role check the role_id attribute of the mongodbatlas_cloud_provider_access resource."

Reply:
Sure, will make changes, thank you!!